recipe/build.sh

```diff
@@ -47,6 +47,15 @@ fi
 
 env | sort
 
+# Disable code signing during the building stage on osx-arm64
+# The code signing in cctools-port is causing the symbolically linked
+# libraries in $PREFIX/lib to become files which causes issues at runtime
+# https://github.com/conda-forge/wxpython-feedstock/issues/74
+# The environment variable disables the step (defined in cctools-port/cctools/include/stuff/port.h)
+# The libraries are still signed during the packaging stage
+if [[ "$CC" == *"arm64"* ]]; then
+  export NO_CODESIGN=1
+fi
```

Comment:
Feedback I got from Isuru on this :

> That should be fine. Better to double check that the package works on osx-arm64
Opening an issue at https://github.com/thefloweringash/sigtool to follow symlinks would be good

@bkpoon I'm assuming you already tested it, right? Can you also open that issue to report the problem to sigtool?

Reply:
Yes, I tested the import from #74 and I tested my own GUI that uses wxPython on Apple Silicon. My GUI does not use every feature of wxPython, though. But I did check the `${CONDA_PREFIX}/lib` directories in the Python 3.8 package for both `osx-64` and `osx-arm64` and they both look similar in terms of symbolic links.

And I'll work on a simple reproducer and create an issue for `sigtool`.

Thanks!

Reply:
Thanks!